packages/oui-navbar/src/index.spec.data.json

```diff
@@ -45,20 +45,35 @@
             "iconClass": "oui-icon oui-icon-info_circle",
             "subLinks": [
               {
-                "title": "Maecenas id ipsum suscipit",
+                "subject": "Maecenas id ipsum suscipit",
+                "time": "10:10",
+                "level": "warning",
                 "url": "#/",
-                "template": "Maecenas id ipsum suscipit, efficitur lectus non, <strong>tempor ante</strong>.",
+                "description": "Maecenas id ipsum suscipit, efficitur lectus non, <strong>tempor ante</strong>.",
                 "isActive": true
-              },
-              {
-                "title": "Vestibulum malesuada justo ac lectus tincidunt",
+              }, {
+                "subject": "Vestibulum malesuada justo ac lectus tincidunt",
+                "time": "yesterday",
+                "level": "info",
                 "url": "#/",
-                "template": "Vestibulum malesuada justo ac lectus tincidunt, <strong>sit amet fringilla lectus iaculis</strong>."
-              },
-              {
-                "title": "Duis placerat lacus at nibh fringilla",
+                "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In fermentum faucibus ante, sed ullamcorper quam pretium ut. Donec et eros at nisi egestas scelerisque quis id erat. Cras suscipit diam id lorem congue aliquam. Integer ornare urna vel turpis blandit, mattis placerat est dictum. Etiam porta venenatis odio tristique condimentum."
+              }, {
+                "subject": "Duis placerat lacus at nibh fringilla",
+                "time": "last week",
+                "level": "success",
+                "url": "#/",
+                "description": "Duis placerat lacus at nibh fringilla, at ultrices nunc suscipit."
+              }, {
+                 "subject": "Mauris ipsum",
+                 "time": "last week",
```

Comment:
"last week" isn't a "time", it's a formatted date

I think we should give a date object to the component and the component is responsible to format it in a standardized way. You could create a custom dateFilter or something like that..

Reply:
moment-angular is not included in the project and I don't think it would be a good idea to do it. User should have control on how date is displayed. "last week" isn't a date either, should I use ```formated_time```?

Reply:
Why moment-angular is a blocker?
You could pass a timestamp or something like that and use native js Date.
IMHO The logic to format it should be at one place, in the component, to be standardized and not duplicated everywhere.